app/src/main/res/values/styles_v2.xml

```diff
@@ -737,4 +737,12 @@
     <item name="shapeAppearanceOverlay">@style/ShapeAppearanceOverlay.Clinic.V2.UnitPreferenceButton</item>
   </style>
 
+  <style name="Clinic.V2.Theme.BottomSheetFragment" parent="Theme.Design.Light.BottomSheetDialog">
```

Comment:
Perhaps we can use the Material styles here?

Reply:
Hmm, not sure which style I should be using. Which one is it?

Reply:
It will be `Theme.MaterialComponents.Light.BottomSheetDialog`. Most of the time we just have to replace `Design` or `AppCompat` to `MaterialComponents`

Reply:
Looks like you forgot to change it for `BottomSheetFragment`

Reply:
NVM, you changed it in next commit.